website/src/lib/oasst_api_client.ts

```diff
@@ -69,7 +69,7 @@ export class OasstApiClient {
     });
 
     if (resp.status === 204) {
-      return null;
+      return null as T;
```

Comment:
I don't really like this, but otherwise we would need to re-write all api function return types